### PR TITLE
Patron Added

### PR DESCRIPTION
**Pull request by Vishwaak**
Issue Id: #114 
 Problem: The  navigation menu did not have a patron link .
 How-to-check: Looking up on the navigation bar in desktop view also in mobile view.

### CODE REVIEW

**Comment by syntaxi** [comment]
Lets preview this

**Comment by syntaxi** [comment]
(testing out a bot thingie)
I'll actual look at this later

**Comment by smsunarto** [comment]
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/121 :rocket:

File: _includes/nav.html [hunk 1]
```diff
@@ -23,6 +23,7 @@
           <li><a href="{{ "/gallery.html" | prepend: site.baseurl }}">Gallery</a></li>
           <li><a href="http://forum.terasology.org">Forum</a></li>
           <li class="green darken-2"><a class="dl-link" href="https://github.com/MovingBlocks/TerasologyLauncher/releases/latest"><i class="material-icons left">cloud_download</i>Get The Game!</a></li>
+          <li class="red lighten-1"> <a href="https://www.patreon.com/Terasology"><img style="height:68%;;"class="img-margin   z-depth-3" src="{{ site.baseurl}}/img/patron.png" alt="You should know"</a></li>
```
Comment (Cervator) [inline _includes/nav.html]: Why is the alt "You should know" ? That's not really a descriptive alternative to an image.

**Review by Cervator — CHANGES_REQUESTED** [review]
Didn't spot this for a while. Not sure the preview is working - the preview for this PR is there still, but shows no sign of the Patreon link. Might just have to preview it locally.

However, I'm not sure why the alt text on an image is "You show know" - that isn't helpful.

File: _includes/nav.html (hunk 1 above)
Comment (Vishwaak) [reply _includes/nav.html]: Sorry my bad, it should have skidded in while debugging. Will get it corrected  by tonight.

Comment (pollend) [reply _includes/nav.html]: you have an extra spaces in your class name. you don't need the semicolon in the style tag.

**Pull request closed without merging**